ToDoOrNotToDo/app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/activity_main"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingBottom="@dimen/activity_vertical_margin"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    tools:context="shuvalov.nikita.todoornottodo.MainActivity">
+
+
+    <LinearLayout
+        android:id="@+id/buttonDisplay"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
+
+        <Button
+            android:id="@+id/addlist"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="Add List"
+            android:background="@color/bluebg"
+            android:textColor="@color/subtext"
+            android:layout_marginBottom="15sp"/>
```

Comment:
Use `dp` here - `sp` is only for text/font sizes - everything else, padding/margin/etc, should be in `dp`